### PR TITLE
Release notes for 4.1

### PR DESCRIPTION
**Pull request by sschaef**
Review by @dragos, @rpokrywka

### CODE REVIEW

**Comment by sschaef** [comment]
@mlangc Thanks for all your work that found its way into this release. I mentioned your name and your Github account, please speak up if you would like to see a link to some other site of yours.

**Comment by mlangc** [comment]
@sschaef Thanks for mentioning me; linking my Github account is perfectly fine!

File: blog/_posts/2015-05-04-release-notes-4.1-RC1.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    11  Iulian Dragos
+    11  Piotr Kukiełka
+     8  Michał Pociecha
+     7  Luc Bourlier
+     6  Krzysztof Romanowski
+     4  Brook Heisler
+     3  François Garillot
+     2  Aleksander Guryanov
+     2  Jason Zaugg
+     2  mpociecha
```
Comment (rpokrywka) [inline blog/_posts/2015-05-04-release-notes-4.1-RC1.md]: I think it's ok to merge manually the count of Michał's commits - 10 Michał Pociecha

File: blog/_posts/2015-05-04-release-notes-4.1-RC1.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+## Thanks
+
+For this release, old contributors moved away, but new ones arrived. Beside from all the work VirtusLab did, special thanks goes to [Matthias Langer][mlangc] who made a great deal in improving this release.
+
+<pre>
+Commits:
+    60  Jerzy Müller
+    57  Matthias Langer
+    44  Simon Schäfer
+    12  Simon Schaefer
```
Comment (rpokrywka) [inline blog/_posts/2015-05-04-release-notes-4.1-RC1.md]: I think it's ok to merge the count manually for You @sschaef :)

**Comment by rpokrywka** [comment]
LGTM

**Pull request merged**